src/js/shield_defs.js

```diff
@@ -3272,18 +3314,35 @@ export function loadShields(shieldImages) {
 
   // OCEANIA
 
-  // New Zealand
-  shields["NZ:SH"] = {
-    backgroundImage: [shieldImages.shield_nz_sh_2, shieldImages.shield_nz_sh_3],
-    textLayoutConstraint: ShieldText.southHalfellipseTextConstraint,
-    textColor: Color.shields.white,
-    padding: {
-      left: 4,
-      right: 4,
-      top: 3,
-      bottom: 5,
-    },
+  // Australia
+  ["ACT", "NSW", "NT", "QLD", "SA", "TAS", "VIC", "WA"].forEach(
+    (state_or_territory) =>
+      ([
+        shields[`AU:${state_or_territory}`],
+        shields[`AU:${state_or_territory}:NH`],
+        shields[`AU:${state_or_territory}:NR`],
+        shields[`AU:${state_or_territory}:S`],
+        shields[`AU:${state_or_territory}:T`],
```

Comment:
Some tourist drives in Queensland are marked by [glyphs](https://www.tmr.qld.gov.au/-/media/busind/techstdpubs/Traffic-management/TC-signs/zipped/q-series.pdf#page=62) rather than numbers:

[![](https://upload.wikimedia.org/wikipedia/commons/thumb/c/c5/Great_Barrier_Reef_Tourist_Drive_road_sign.jpg/1024px-Great_Barrier_Reef_Tourist_Drive_road_sign.jpg)](https://commons.wikimedia.org/wiki/File:Great_Barrier_Reef_Tourist_Drive_road_sign.jpg)

Some of these relations have been tagged with keyword-like `ref` values, such as [`ref=dugong`](https://www.openstreetmap.org/relation/13493955), based on [a table](https://wiki.openstreetmap.org/wiki/Custom_Highway_Shields#Australia) that’s been on the wiki for a long time. I guess we should exclude them for now, and when we’re able to design copyright-unencumbered artwork for them, we can add a manual mapping from `ref`s or `name`s to images.

Reply:
Seems like a similar situation to the [Touring Routes of New Zealand,](https://github.com/ZeLonewolf/openstreetmap-americana/issues/451#issuecomment-1169355237) each of which has unique artwork. The solution in that case was to adopt guidelines that tag each route with a unique value of `network` matching `NZ:Touring:*` so that each one can be assigned a different icon. Emojis were chosen as `ref` values as a fallback for less sophisticated renderers, though this wasn't strictly necessary.

@DianAgesson would Queensland mappers be opposed to assigning unique `network` values for their non-numeric Tourist Drives? Even though they may all belong to one "network" in reality, the shield appearance is different enough that using a generic `network` value would produce misleading renderings.

Reply:
Oh no, there are always edge cases that haven’t been accounted for aren’t there! What’s wrong with using numbers?! :)

I’ll reach out to the active mappers on discord/mailing list. I should be able to update the tagging guidelines for these shields accordingly.

Reply:
Hello,

Just updating you with our progress.

The tagging guidelines have been adjusted for these edge cases. I can see you’ve already updated for the State Strategic Touring Routes, and if there are other cases that come up I’ll pass it on.

Bulk editing has also kicked off in earnest. Lots of contributors have been chipping away in places as well; WA is completely converted, and NT and TAS have transitioned their relations.

Dian

Reply:
A large majority of the route relations have now been converted. Two previously undocumented route types where identified.

Overdimensional routes (eg AU:VIC:OD), and freight routes (eg AU:QLD:Freight). Neither type of route requires a shield to be rendered.

It'll take more time to progressively update the highway road segments, but my understanding is that the shield rendering uses the relation anyway?

Reply:
> It'll take more time to progressively update the highway road segments, but my understanding is that the shield rendering uses the relation anyway?

Correct, this style doesn’t consider way refs at all. This means most of the road network is ready for the vector tiles to be regenerated.

Reply:
Sounds good, we're overdue for a tile update anyways.